### PR TITLE
DHFPROD-4830: Check for false positives on search for multiple merge …

### PR DESCRIPTION
**Pull request by ryanjdew**
…actions on document

#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mastering/default/merging/falsePositivesForOtherMerges.sjs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const merging = require("/data-hub/5/builtins/steps/mastering/default/merging.sjs");
+const test = require("/test/test-helper.xqy");
+
+const assertions = [];
+let caughtException = null;
+try {
+    merging.main({uri: '/merge-with-doc1.json'}, {mergeOptions: {}});
```
Comment (rjrudin) [inline marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mastering/default/merging/falsePositivesForOtherMerges.sjs:7]: I think there should be an assertion here on the results of this, because shouldn't those be different now? I Otherwise, the reader has to analyze the test documents to figure out the scenario and why things are working now, which I wasn't able to do.

Comment (ryanjdew) [reply marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mastering/default/merging/falsePositivesForOtherMerges.sjs:7]: No, the results won't differ, really. The only difference that is patch is making is that in one corner case it will no longer throw an exception.

It was hitting a null pointer error due to a false positive when looking for another match summary document that has other merge actions for the same document.
